src/routes/events/containers/EventsPage.js

```diff
@@ -51,15 +51,15 @@ export class EventsPage extends React.Component<Props> {
   }
 }
 
-const mapStateToProps = (state: State) => ({
-  language: state.location.payload.language,
-  city: state.location.payload.city,
-  eventId: state.location.payload.eventId,
-  events: state.events.data,
-  cities: state.cities.data
+const mapStateTypeToProps = (stateType: StateType) => ({
+  language: stateType.location.payload.language,
+  city: stateType.location.payload.city,
+  eventId: stateType.location.payload.eventId,
+  events: stateType.events.data,
+  cities: stateType.cities.data
```

Comment:
`stateType` looks odd, as its not a type. Should be `state`.

Reply:
Yep, thank :+1: